### PR TITLE
Add docstrings to fab tasks to make them more understandable

### PR DESCRIPTION
**Pull request by sevenmachines**
Add in some more documentation into the fab tasks

### CODE REVIEW

File: bootstrap_cfn/fab_tasks.py [hunk 1]
```diff
@@ -37,40 +37,124 @@
...
 @task
 def aws(x):
+    """
+    Set the AWS account to use
+
+    Sets the environment variable 'aws' to the name of the
+    account to use in the AWS config file (~/.config.yaml)
+
+    Args:
+        x(string): The string to set the environment
```
Comment (ashb) [inline bootstrap_cfn/fab_tasks.py]: Lets change the name of the parameter from `x` to `profile_name`

File: bootstrap_cfn/fab_tasks.py [hunk 2]
```diff
@@ -37,40 +37,124 @@
 
 @task
 def aws(x):
+    """
+    Set the AWS account to use
+
+    Sets the environment variable 'aws' to the name of the
+    account to use in the AWS config file (~/.config.yaml)
```
Comment (ashb) [inline bootstrap_cfn/fab_tasks.py]: These actually look in ~/.aws/credentials.yaml

File: bootstrap_cfn/fab_tasks.py [hunk 3]
```diff
@@ -37,40 +37,124 @@
...
+        x(string): The string to set the
+        variable to
+    """
     env.application = str(x).lower()
 
 
 @task
 def config(x):
+    """
+    Set the location of the projects YAML file
```
Comment (ashb) [inline bootstrap_cfn/fab_tasks.py]: "project's" please.

File: bootstrap_cfn/fab_tasks.py [hunk 4]
```diff
@@ -37,40 +37,124 @@
...
     env.user = x
 
 
 def get_stack_name():
+    """
+    Get the name of the stack
+
+    The name of the stack is a combination
+    of the application and environment names
+    """
```
Comment (ashb) [inline bootstrap_cfn/fab_tasks.py:157]: Not a fab tasks but docs are good :D

**Comment by sevenmachines** [comment]
Text changes made. README updated with new aws config.yaml location

**Comment by ashb** [comment]
Good work!

**Pull request merged**